src/main/java/io/openepcis/convert/collector/JsonEPCISEventCollector.java

```diff
@@ -118,9 +123,9 @@ public void end() {
       jsonGenerator.writeEndObject(); // End epcisBody
 
       // Write the info related to Context element in JSON
-      jsonGenerator.writeFieldName("@context");
+      jsonGenerator.writeFieldName(EPCIS.CONTEXT);
       jsonGenerator.writeStartArray();
-      jsonGenerator.writeString("https://ref.gs1.org/standards/epcis/2.0.0/epcis-context.jsonld");
+      jsonGenerator.writeString(EPCIS.DEFAULT_CONTEXT);
```

Comment:
We might want to move the default jsonld @context from EPCIS.DEFAULT_CONTEXT to EPCISVersion.getDefaultJSONContext